app/routes.py

```diff
@@ -1,2 +1,328 @@
-from flask import Blueprint
+from app import db
+from app.models.customer import Customer
+from app.models.video import Video
+from app.models.rental import Rental
+from datetime import datetime
+from flask import Blueprint, request, jsonify
+from flask.helpers import make_response
```

Comment:
these can be merged into one!

```suggestion
from flask import Blueprint, request, jsonify, make_response
```